### DISCUSSION TITLE
RFID - Card Action script

### DISCUSSION THREAD

**Discussion by jensbab** [MiczFlor/RPi-Jukebox-RFID#2047] [Q&A]
Hi Community,
first of all wanna say thank You for great opensource projects like the RFID-Jukebox :-)

I was looking around to find an Option in the corresponding config file for RFID Triggers. But couldnt find. Actually i am building an Box for my kids and i want to include some smart home things like starting a script when a card was swiped.

I want to start a playlist and at the end i want to turn of the light in the room. To turn off the light i just need an Call to a website.

Is this possible somehow?

**Comment by ConsolasTrovao** [comment]
This is a really cool idea. Maybe digging around the read.py script? I am just starting to use this so I don't know where the "core" code is.

**Reply by jensbab** [reply]
i think the clue is to add like an CMD but actually not found where this might be added

**Reply by jensbab** [reply]
seems the rfid_trigger_play.sh is it

**Reply by ConsolasTrovao** [reply]
Nice! Did it work? Will check it as well - curious about this.

**Reply by jensbab** [reply]
i must update the code accordingly. i will do that later. now at work its not the time and need a bit to concentrate :-)

**Reply by ConsolasTrovao** [reply]
Ahah of course! Keep us posted!

**Comment by AlvinSchiller** [comment]
Hi, have you had a look if the mqtt integration fits your case?
https://github.com/MiczFlor/RPi-Jukebox-RFID/wiki/Smart-Home-remote-control-with-MQTT

Haven't tried it myself though.
